content/sensu-go/5.17/guides/email-handler.md

```diff
@@ -0,0 +1,311 @@
+---
+title: "Send email alerts with the Sensu Go Email Handler"
+linkTitle: "Send Email Alerts"
+description: "Here’s how to send alerts to your email with the Sensu Go Email Handler. Use handlers to send events to your technology of choice (in this case, email) to alert you of incidents and help you resolve them more quickly."
+weight: 85
+version: "5.17"
+product: "Sensu Go"
+platformContent: False
+menu: 
+  sensu-go-5.17:
+    parent: guides
+---
+
+- [Create an ad hoc event](#create-an-ad-hoc-event)
+- [Add the email handler asset](#add-the-email-handler-asset)
+- [Add an event filter](#add-an-event-filter)
+- [Assign the email handler to a check](#assign-the-email-handler-to-a-check)
+- [Create an email template](#create-an-email-template)
+- [Trigger an event](#trigger-an-event)
+- [Next steps](#next-steps)
+
+Sensu event handlers are actions the Sensu backend executes on [events][1].
+This guide explains how to use the [Sensu Go Email Handler][3] to send notification emails.
+
+To follow this guide, you’ll need to [install the Sensu backend][12] and have at least one [Sensu agent][13] running on Linux.
+You should also [install and configure sensuctl][4].
+
+## Create an ad hoc event
```

Comment:
We don't need to create this event before we configure the handler. I think this section should be merged with the `Trigger an event` section.